assets/js/components/DataBlockGroup.js

```diff
@@ -60,14 +59,17 @@ export default function DataBlockGroup( { className, children } ) {
 				// Calculate the exact scale factor needed to resize the content to the parent.
 				const scaleFactor = parentWidth / dataPoint.scrollWidth;
 
-				if ( scaleFactor < smallestScaleFactor ) {
-					smallestScaleFactor = scaleFactor;
+				// Round scaling factor down 1D.P to account for variations in font rendering causing inconsistent resizing for VRTs.
```

Comment:
I would prefer not to see anything in the production code which is specifically there just to make tests pass.

If we can justify this change as a general improvement in font rendering stability for the component, that would be preferable.